lib/rubycritic/platforms/darwin.rb

```diff
@@ -0,0 +1,29 @@
+require "rubycritic/platforms/base"
+
+module Rubycritic
+  module Platforms
+    class Darwin < Base
+      def can_open?
+        platform =~ /darwin/
+      end
+
+      def browser_hash
```

Comment:
No hungarian notation please. How about "browser_map"? ("map" is some kind of generic interface while "hash" is a concrete type).